lib/Froxlor/Cron/Http/Php/Fcgid.php

```diff
@@ -185,22 +185,26 @@ public function createIniFile($phpconfig)
 			$openbasedirc = ';';
 		}
 
-		$admin = $this->getAdminData($this->domain['adminid']);
-		$php_ini_variables = [
-			'SAFE_MODE' => 'Off', // keep this for compatibility, just in case
-			'PEAR_DIR' => Settings::Get('system.mod_fcgid_peardir'),
-			'TMP_DIR' => $this->getTempDir(),
-			'CUSTOMER_EMAIL' => $this->domain['email'],
-			'ADMIN_EMAIL' => $admin['email'],
-			'DOMAIN' => $this->domain['domain'],
-			'CUSTOMER' => $this->domain['loginname'],
-			'ADMIN' => $admin['loginname'],
-			'OPEN_BASEDIR' => $openbasedir,
-			'OPEN_BASEDIR_C' => $openbasedirc,
-			'OPEN_BASEDIR_GLOBAL' => Settings::Get('system.phpappendopenbasedir'),
-			'DOCUMENT_ROOT' => FileDir::makeCorrectDir($this->domain['documentroot']),
-			'CUSTOMER_HOMEDIR' => FileDir::makeCorrectDir($this->domain['customerroot'])
-		];
+$admin = $this->getAdminData($this->domain['adminid']);
+
+$admin_email = (is_array($admin) && isset($admin['email'])) ? $admin['email'] : '';
+$admin_loginname = (is_array($admin) && isset($admin['loginname'])) ? $admin['loginname'] : '';
+
+$php_ini_variables = [
+    'SAFE_MODE' => 'Off',
+    'PEAR_DIR' => Settings::Get('system.mod_fcgid_peardir'),
+    'TMP_DIR' => $this->getTempDir(),
+    'CUSTOMER_EMAIL' => $this->domain['email'],
+    'ADMIN_EMAIL' => $admin_email,
```

Comment:
save an unnecessary variable and if not used elsewhere, directly do
```
'ADMIN_EMAIL' => $admin['email] ?? '',
```

same with admin-loginname